tweetapi/login.py

```diff
@@ -91,17 +92,17 @@ class Resetpassword(CreateAPIView):
     serializer_class = SendUserPasswordReset
     def post(self, request):
         serializer = self.get_serializer(data=request.data)
-        if serializer.is_valid():
-            return Response({'message':"password reset sent, check your mail"}, status=status.HTTP_200_OK)
-        return Response(serializer.errors,status=status.HTTP_400_BAD_REQUEST)
+        serializer.is_valid(raise_exception=True)
+        return Response({'message':"password reset sent, check your mail"}, status=status.HTTP_200_OK)
+    
 
 
 class UserPasswordReset(CreateAPIView):
     serializer_class = UserPasswordReset
     def post(self, request,uid,token,format=None):
         serializer = self.get_serializer(data=request.data,
         context={'uid':uid,'token':token})
-        if serializer.is_valid(raise_exception=True):
+        if serializer.is_valid():
```

Comment:
why not raise exception?

Reply:
in all honesty sir i thought i called the raise exception didnt know it was just Resetpassword